packages/frontend-shared/src/locales/en-US.json

```diff
@@ -94,8 +94,8 @@
         "specsAddingHeader": "Adding specs..."
       },
       "importEmptySpec": {
-        "header": "Create new empty spec",
-        "description": "We'll generate an empty spec file which can be used to start testing your application.",
+        "header": "Create starter spec",
```

Comment:
```suggestion
        "header": "Create new spec",
```

Reply:
Hello @emilyrohrbough, I have submitted another pull request to address the Contributor License Agreement (CLA) issue.

In response to the comments on the PR, I have used the term 'starter' instead of 'new spec' as suggested in this issue comment (https://github.com/cypress-io/cypress/issues/21908#issuecomment-1154792807). 

Can you please share your thoughts on this? Thank you.